CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+# [0.60.0] - 2024-08-23
+### Changed
+- `seq add` commands now sends a hash of the file to the server for verification.
```

Comment:
Would you mind updating to "`seq add` commands now sends a hash of the file to the server for verification, rather than checking it after upload." ... just to be politic (i.e. there was not no hash check previously)